scripts/wasmd/deploy_cw3.js

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
```

Comment:
Can you delete the deployment scripts (and contracts) for now? They cannot work before https://github.com/cosmos/cosmjs/issues/491